src/Microsoft.AspNetCore.JsonPatch/Adapters/IObjectAdapter.cs

```diff
@@ -15,5 +15,6 @@ public interface IObjectAdapter
         void Move(Operation operation, object objectToApplyTo);
         void Remove(Operation operation, object objectToApplyTo);
         void Replace(Operation operation, object objectToApplyTo);
+        void Test(Operation operation, object objectToApplyTo);
```

Comment:
@eilon, this is a breaking change.

The non-breaking way to do this is to introduce a base-class or another interface and conditionally cast to that.

Reply:
Yeah a 2nd interface might be the best choice here. Too bad we got it wrong in v1 😦